pkg/scheduler/algorithm/predicates/metadata.go

```diff
@@ -690,13 +678,14 @@ func getAffinityTermProperties(pod *v1.Pod, terms []v1.PodAffinityTerm) (propert
 		return properties, nil
 	}
 
-	for _, term := range terms {
-		namespaces := priorityutil.GetNamespacesFromPodAffinityTerm(pod, &term)
+	for i := range terms {
+		term := &terms[i]
```

Comment:
nit: leave the for as it was

Reply:
changing to use topologyKey though allows us to revert back to using the old approach.